common_test.go

```diff
@@ -0,0 +1,27 @@
+package common
+
+import (
+	"net/http"
+	"testing"
+
+	"github.com/stretchr/testify/require"
+	"github.com/zeebo/errs"
+)
+
+var statusFromErrorTests = []struct {
+	err      error
+	expected int
+}{
+	{ServerError.New("server"), http.StatusInternalServerError},
```

Comment:
i'd leave the strings being passed to New empty on these so there is no confusion on whether they have an impact on the status returned from statusFromError